src/Action/NullAction.php

```diff
@@ -0,0 +1,25 @@
+<?php
+
+/*
+ * This file is part of the API Platform project.
+ *
+ * (c) Kévin Dunglas <dunglas@gmail.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace ApiPlatform\Core\Action;
+
+/**
+ * Empty action. Useful to trigger the kernel.view event without doing anything specific in the action
```

Comment:
Why not go all the way and remove all other actions? :laughing:

So we will end up with an event listener which fetches data from the data provider, and another which performs the deserialization (if applicable).